addons/binding/org.openhab.binding.sleepiq/src/main/java/org/openhab/binding/sleepiq/config/SleepIQCloudConfiguration.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.sleepiq.config;
+
+/**
+ * Configuration class for the SleepIQ cloud.
+ *
+ * @author Gregory Moyer - Initial contribution
+ */
+public class SleepIQCloudConfiguration {
+
+    public static final String USERNAME = "username";
+    public static final String PASSWORD = "password";
+    public static final String POLLING_INTERVAL = "pollingInterval";
+    public static final String LOGGING = "logging";
```

Comment:
1) I agree, i might have searched for constant instead of the field
2) Personally I would always enable the logging in the 'external' framework and base the activeness of logging on log levels. 

You might solve this by passing `logger.isDebugEnabled()` of the binding classes which will be true if debug logging is enabled for that class, in that way you don't require configuration but can enable / disable it like any other openHAB logging.